From 7a47e41ce8c20408f2805f9e93631f0000bea560 Mon Sep 17 00:00:00 2001
From: Alex Deucher <alexdeucher@gmail.com>
Date: Tue, 27 Jul 2010 11:01:15 -0400
Patch-mainline: 2.6.36
References: fate#310916
Git-commit: b417cc117a104ea457be18cfde9df9d69c21fa62
Subject: [PATCH 0982/2588] drm/radeon: remove viewport transform from
 r6xx/r7xx blit emit

We aren't using it, so no need.  Save additional dwords.

Signed-off-by: Alex Deucher <alexdeucher@gmail.com>
Signed-off-by: Dave Airlie <airlied@redhat.com>
(cherry picked from commit b417cc117a104ea457be18cfde9df9d69c21fa62)

Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 drivers/gpu/drm/radeon/r600_blit_shaders.c |   18 ------------------
 1 files changed, 0 insertions(+), 18 deletions(-)

diff --git a/drivers/gpu/drm/radeon/r600_blit_shaders.c b/drivers/gpu/drm/radeon/r600_blit_shaders.c
index 762b81c..e8151c1 100644
--- a/drivers/gpu/drm/radeon/r600_blit_shaders.c
+++ b/drivers/gpu/drm/radeon/r600_blit_shaders.c
@@ -127,15 +127,6 @@ const u32 r6xx_default_state[] =
 	0x00000000, /* DB_STENCILREFMASK_BF */
 	0x00000000, /* SX_ALPHA_REF */
 
-	0xc0066900,
-	0x0000010f,
-	0x00000000, /* PA_CL_VPORT_XSCALE */
-	0x00000000,
-	0x00000000,
-	0x00000000,
-	0x00000000,
-	0x00000000,
-
 	0xc0046900,
 	0x0000030c,
 	0x01000000, /* CB_CLRCMP_CNTL */
@@ -455,15 +446,6 @@ const u32 r7xx_default_state[] =
 	0x00000000, /* DB_STENCILREFMASK_BF */
 	0x00000000, /* SX_ALPHA_REF */
 
-	0xc0066900,
-	0x0000010f,
-	0x00000000, /* PA_CL_VPORT_XSCALE */
-	0x00000000,
-	0x00000000,
-	0x00000000,
-	0x00000000,
-	0x00000000,
-
 	0xc0046900,
 	0x0000030c, /* CB_CLRCMP_CNTL */
 	0x01000000,
-- 
1.7.6

